lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -431,10 +433,11 @@ llvm::Error Caffe2ModelLoader::loadConvQuantized(const caffe2::OperatorDef &op,
   // C - output_depth, R - filter_height, S - filter_width, K - input_depth.
   // For Caffe2 "Int8Conv" and "Int8ConvRelu", the weights always follows the
   // "order" arg.
-  if (order != "NHWC") {
+  if (order != GLOW_DEFAULT_4D_LAYOUT) {
```

Comment:
Please use a concrete string here, because any change to the definition of GLOW_DEFAULT_4D_LAYOUT would break this code.